sites/partners/src/pages/listings/[id]/applications/index.tsx

```diff
@@ -53,6 +53,8 @@ const ApplicationsList = () => {
   const countyCode = listingDto?.jurisdictions.name
   const listingName = listingDto?.name
   const isListingOpen = listingDto?.status === "active"
+  const isListingClosed = listingDto?.status === "closed"
+  const isListingClosedAdmin = isListingClosed && profile?.userRoles?.isAdmin
```

Comment:
Nit but I think this could be simplified to 
const allowNewPaperApps = listingDto?.status !== "closed" || profile?.userRoles?.isAdmin